modules/virt-about-the-overview-dashboard.adoc

```diff
@@ -6,6 +6,13 @@
 [id="virt-about-the-overview-dashboard_{context}"]
 = About the {product-title} dashboards page
 
+Access the {product-title} dashboard, which captures high-level information
+about the cluster, by navigating to *Home* -> *Overview* from
+the {product-title} web console.
+
+The {product-title} dashboard provides various cluster information, captured in
+individual dashboard cards.
```

Comment:
*PEER REVIEW* Moved this text unchanged [from the assembly](https://github.com/openshift/openshift-docs/pull/67647/files#diff-cee282cef256440027d3f0756d83e4bae8eef3a965783f31f194e2f8146651c8L9).